src/App.jsx

```diff
@@ -1,8 +1,7 @@
 
 import { useState } from 'react'
-import { BrowserRouter } from "react-router-dom";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 import Nav from "./assets/components/Nav/Nav";
-import Allbooks from './Components/HomePage/AllBooks/AllBooks'
 import HomePage from './Components/HomePage/HomePage'
```

Comment:
This line should be removed as well

Reply:
We have already done this one.  Just re-checked the file and its not there.